src/components/CardList.jsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import Card from "./Card";
 
 const CardList = ({ robos }) => {
+  console.log("Render Card List");
```

Comment:
I think this was mistakenly left in the code.

Reply:
Thanks, will correct it in next commit.